### ISSUE TITLE
Layer Wise Learning Rate 

### ISSUE THREAD

**Issue by jrock08** [aiUIUC/pyAIUtils#35]
I think something like: 

`multi_rate_optimizer(optimizer, learning_rate_list, variables_list, loss)`

That returns a properly canned opt which computes gradients and applies them could be useful.

Eg code: http://stackoverflow.com/questions/34945554/how-to-set-layer-wise-learning-rate-in-tensorflow

**Comment by jrock08** [comment]
See Pull request #37
